php/libraries/Visit.class.inc

```diff
@@ -0,0 +1,220 @@
+<?php
+/**
+ * This file contains the Visit class.
+ *
+ * PHP Version 5
+ *
+ * @category Main
+ * @package  Main
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+/**
+ * Base class that contains method to access the visit of projects
+ *
+ * @category Main
+ * @package  Main
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class Visit
+{
+    /**
+     * Return an array containing the label and legacyLabel of a visit
+     *
+     * @param integer $id the identifier of the label
+     *
+     * @throws DatabaseException
+     *
+     * @return array a hash with both label
+     *                 hash is of the form
+     *                 array('Label'       => 'label',
+     *                       'LegacyLabel' => 'legacy label')
+     */
+    public function getLabel($id)
+    {
+        $DB = Database::singleton();
+
+        $result = $DB->pselect(
+            "SELECT VisitLabel, VisitLegacyLabel
+             FROM visit
+             WHERE VisitID = :id",
+            array('id' => $id)
+        );
+        return $result;
+    }
```

Comment:
As a model, these should be injected into the constructor, not selected from the database. The database access should be decoupled.